projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -5,6 +5,14 @@ $box-shadow-color: rgba( 0, 0, 0, 0.1 );
 .container {
 	position: relative;
 	min-height: 200px;
+	&.has-error {
+		border-width: 2px;
+		border-color: var( --jp-red-50 );
+	}
+	&.has-warning {
+		border-width: 2px;
+		border-color: #DEB100;
+	}
```

Comment:
I saw that the styles show that the borders are 2px wide, but I personally think that 1px looks better and less aggressive, even though 2px draws more attention to the cards:

2px | 1px
--- | --- 
<img width="368" alt="image" src="https://github.com/user-attachments/assets/b6fe4163-ff29-44e7-b401-8116797bc396">|<img width="368" alt="image" src="https://github.com/user-attachments/assets/37020750-9c4a-43bd-9b19-a1c29b29f007">
<img width="1095" alt="image" src="https://github.com/user-attachments/assets/f4b84b3c-6774-4490-87e8-ccc78857d0b2">|<img width="1095" alt="image" src="https://github.com/user-attachments/assets/0065e946-48da-48a7-a639-3d54ee8493ad">

Reply:
Ok yeah, sounds good. Thanks Ian, I changed the error & warning border to 1px. 👍